documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -97,11 +99,12 @@ There are two ways of adding a Trait:
 * Drag and drop one from the Workspace sidebar
 * Click the top Add icon at a Characteristic
 
-Clicking the top Add icon image:add.png[Add icon] of a Characteristic will add a new Trait, taking the Characteristic's position and connecting the Trait to the initial Characteristic as well as a newly created Constraint. In this scenario, the newly created Trait takes the name of the parent attribute as a prefix to its own name.
+Clicking the top Add icon image:add.png[Add icon] of a Characteristic will add a new Trait, taking the Characteristic's position and connecting the Trait to the initial Characteristic as well as a newly created Constraint.
+In this scenario, the newly created Trait takes the name of the parent attribute as a prefix to its own name.
 
 A Trait has an Add icon image:add.png[Add icon] on the bottom of the element, which is always visible:
 
-* When clicking the Add icon image:add.png[Add icon], a Characteristic is created and connected to the Trait. 
+* When clicking the Add icon image:add.png[Add icon], a Characteristic is created and connected to the Trait.
```

Comment:
When clicking the "Add" icon image:add.png[Add icon], a Characteristic is created and connected to the Trait.